decidim-core/app/uploaders/decidim/image_uploader.rb

```diff
@@ -11,9 +11,7 @@ class ImageUploader < ApplicationUploader
     # CarrierWave automatically calls this method and validates the content
     # type fo the temp file to match against any of these options.
     def content_type_whitelist
-      [
-        %r{image\/}
-      ]
+      extension_whitelist.map { |ext| "image/#{ext}" }
```

Comment:
There are some extension from the whitelist that aren't images maybe we need to add extension and mimes somewhere?

Reply:
Oh, forget about this, I read the list from the attachment uploader, not the image!